{"name": "Попытка войти в систему с несуществующими данными", "status": "passed", "description": "Входим в систему с невалидными данными и получаем ошибку", "steps": [{"name": "Попытка залогиниться в систему с несуществующими логином и паролем", "status": "passed", "start": 1721062853795, "stop": 1721062853976}], "start": 1721062853795, "stop": 1721062853976, "uuid": "2668a1a2-5869-4b54-b791-8ec79d1b492c", "historyId": "34c9ed1761f73b4da9b6c27596000850", "testCaseId": "34c9ed1761f73b4da9b6c27596000850", "fullName": "tests.test_login_courier.TestLoginCourierAPI#test_login_invalid_data", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestLoginCourierAPI"}, {"name": "host", "value": "DESKTOP-LQ0DQPC"}, {"name": "thread", "value": "17932-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}]}